packages/patternfly-4/react-table/src/components/Table/Table.d.ts

```diff
@@ -48,22 +48,27 @@ export interface IExtra extends IExtraData {
 }
 
 export interface ISortBy {
-  index?: Number;
+  index?: number;
   direction?: OneOf<typeof SortByDirection, keyof typeof SortByDirection>;
 }
 
 export interface IAction {
-  title: String;
+  title: string;
   onClick: (event: MouseEvent, rowIndex: number, rowData: IRowData, extraData: IExtraData) => void;
 }
 
 export interface ISeparator {
   isSeparator: Boolean
 }
 
+export interface ITransform {
```

Comment:
These functions can return object that contains eny HTML property so extendind `HTMLProps` is a good way. And probably renaming it to `IDecorator` so we can reuse this interface in many places?
```TSX
export interface ITransform extends HTMLProps<HTMLElement>{
  isVisible: boolean;
  // children doesn't have to be required
}
```